Source/relay/TourGuide/Items of the Month/2021/Daylight Shavings Helmet.ash

```diff
@@ -1,93 +1,90 @@
 //Daylight Shavings Helmet
+
+string [int, string] dshBuffOrder = {
```

Comment:
this is a slick and markedly better way to achieve the result of the tooltip table below. good work.

Reply:
👍 Doing this in ASH is a little weird compared to TS, but I really wanted to get rid of the duplication, switch/case, etc and this direction ended up working out okay